Changelog.txt

```diff
@@ -28,7 +28,7 @@ Bug fixes:
 - SCP-173's collision radius has been decreased.
 - Fixed a crash that occured when opening your inventory.
 - The fog doesn't turn white anymore if you go to the pocket dimension by leaving SCP-1499.
-- Fixed items and npcs falling through the floor if the room is not rendered.
+- Fixed items,npcs and the player not falling through the floor anymore if the room is not rendered.

```

Comment:
Two minor suggestions: there's no need to keep instantiating and freeing those pivots, you could just do a linepick straight down. I'd also rather call it PickDistance instead of PickRadius, because the variable doesn't represent any sort of radius.